interface/ceed-operator.c

```diff
@@ -91,20 +97,32 @@ static int CeedOperatorCheckField(Ceed ceed, CeedQFunctionField qf_field, CeedEl
 
   @ref Utility
 **/
-static int CeedOperatorFieldView(CeedOperatorField field, CeedQFunctionField qf_field, CeedInt field_number, bool sub, bool input, FILE *stream) {
-  const char *pre    = sub ? "  " : "";
-  const char *in_out = input ? "Input" : "Output";
+static int CeedOperatorFieldView(CeedOperatorField op_field, CeedQFunctionField qf_field, CeedInt field_number, bool sub, bool input, FILE *stream) {
+  const char  *pre    = sub ? "  " : "";
+  const char  *in_out = input ? "Input" : "Output";
+  char        *field_name;
+  CeedInt      size;
+  CeedEvalMode eval_mode;
+  CeedVector   vec;
+  CeedBasis    basis;
+
+  // Field data
+  CeedCall(CeedQFunctionFieldGetName(qf_field, &field_name));
+  CeedCall(CeedQFunctionFieldGetSize(qf_field, &size));
+  CeedCall(CeedQFunctionFieldGetEvalMode(qf_field, &eval_mode));
+  CeedCall(CeedOperatorFieldGetVector(op_field, &vec));
+  CeedCall(CeedOperatorFieldGetBasis(op_field, &basis));
```

Comment:
Thoughts on an "unpack" function to (optionally) get these values out, with `NULL` being used to denote non-desired values?

Reply:
Yeah, that seems reasonable

Reply:
I see other places where that same idea could be used, but "unpack" gets worse with increasing number of parameters to unpack. Like, there are a few 10 successive lines of `CeedBasisGet*`, but I feel like a 10+ parameter unpack function could be unreadable at some point, especially with `NULL` pass throughs.

Reply:
See https://github.com/CEED/libCEED/pull/1483/commits/ab74770665870aeb8f4af2e1fe7995e7c4c2ce73

This will also be helpful in Ratel